terraform/modules/route53/variables.tf

```diff
@@ -0,0 +1,111 @@
+variable "root_domain_name" {
+  type        = string
+  description = "The root domain name for DROITS"
+}
+
+variable "create_hosted_zone" {
+  description = "Flag to create hosted zone"
+  type        = bool
+}
+
+variable "production_webapp_alb_dns" {
+  description = "The DNS name of the production webapp ALB"
+  type        = string
+}
+
+variable "production_backoffice_alb_dns" {
+  description = "The DNS name of the production backoffice ALB"
+  type        = string
+}
+
+variable "staging_webapp_alb_dns" {
+  description = "The DNS name of the staging webapp ALB"
+  type        = string
+}
+
+variable "staging_backoffice_alb_dns" {
+  description = "The DNS name of the staging backoffice ALB"
+  type        = string
+}
+
+variable "dev_webapp_alb_dns" {
+  description = "The DNS name of the dev webapp ALB"
+  type        = string
+}
+
+variable "dev_backoffice_alb_dns" {
+  description = "The DNS name of the dev backoffice ALB"
```

Comment:
It's a bit much having so many variables here. We can group these together. Given we're only going to deploy the hosted zone to prod I reckon we should create the prod records explicitly (perhaps populated by values from the other modules.) but have an unknown number of other records that get created?